_fr/tour/abstract-type-members.md

```diff
@@ -2,11 +2,80 @@
 layout: tour
 title: Abstract Type Members
 partof: scala-tour
-
-num: 21
-
+num: 25
 language: fr
-
 next-page: compound-types
 previous-page: inner-classes
+topics: abstract type members
+prerequisite-knowledge: variance, upper-type-bound
+
+redirect_from: 
+  - "/tutorials/tour/abstract-types.html"
+  - "/tour/abstract-types.html"
```

Comment:
I guess these shouldn't be here? Or they should refer to the `fr` versions of those URLs?